README.adoc

```diff
@@ -42,37 +45,55 @@ and configure a _Run/Debug configuration_ like this:
 * Working directory: `$MODULE_DIR$` (must resolve to optaplanner-examples directory)
 * Use classpath of module: `optaplanner-examples`
 
-=== Starter issues
+### Starter issues
 
 If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+## Contributing to OptaPlanner
+
+This is an open source project, and you are more than welcomed to contribute!:heart:
```

Comment:
I think that "welcome" is an adjective in this sentence:

```suggestion
This is an open-source project, and you are more than welcome to contribute!:heart:
```

Reply:
Can we add a space before the :heart: ?

Reply:
Sure, I was planning to invite @emmurphy1 as well